django_cloud_tasks/tasks/task.py

```diff
@@ -182,9 +182,20 @@ def sync(cls, **kwargs):
         return cls().run(**kwargs)
 
     @classmethod
-    def asap(cls, **kwargs):
-        return cls.push(task_kwargs=kwargs)
-
+    def asap(
+        cls,
+        headers: dict | None = None,
+        queue: str | None = None,
+        delay_in_seconds: int | None = None,
+        **kwargs,
+    ):
+        return cls.push(
+            task_kwargs=kwargs,
+            queue=queue,
+            headers=headers,
+            delay_in_seconds=delay_in_seconds,
+        )
+        
```

Comment:
This library is heavily inspired in Celery to provide a familiar interface for projects migration from Celery to GCP serverless.
The calling signature is one of the most important parts, and Celery has 2 methods:
- a fully-featured method that supports execution parameters (such as routing, delaying): `apply_async`
- a sugar-syntax method that just calls a task: `delay`

We want to provide the same features here: an easy-but-limited signature and a complex-but-featured signature.
The `asap` is the easy-and-limited.
The `push` is the complex-but-featured.

By accepting `headers`, `queue` and `delay_in_seconds` in the `asap`, you're not only breaking the familiarity, but you're eventually restricting arguments for the `asap`
For instance: if I had a task that accepts a argument named queue by coincidence, and I'm using `asap`, after this change, my task won't get this argument anymore, because you're absorbing it and considering it as the routing queue.

My suggestion is: stick to `push` for complex calls and leave the `asap` (and other friendlier methods) for simple interface.
You can always extend the `Task` class in your own project and override it there, if you will.

Reply:
But adding the queue paramenter to asap looks a very nice feature